rasa/core/policies/ted_policy.py

```diff
@@ -534,24 +577,26 @@ def train(
         model_data = self._create_model_data(
             tracker_state_features, label_ids, entity_tags, encoded_all_labels
         )
-        if model_data.is_empty():
-            logger.error(
-                f"Can not train '{self.__class__.__name__}'. No data was provided. "
-                f"Skipping training of the policy."
-            )
-            return
```

Comment:
Why do we no longer need this?

Reply:
Oh actually, it was moved into `train`, yes?

Reply:
Yes.